centreon/tests/e2e/features/Dashboards/10-widget-metric-graph-configuration/index.ts

```diff
@@ -0,0 +1,395 @@
+import { Given, When, Then } from '@badeball/cypress-cucumber-preprocessor';
+
+import { PatternType } from '../../../support/commands';
+import dashboardAdministratorUser from '../../../fixtures/users/user-dashboard-administrator.json';
+import dashboards from '../../../fixtures/dashboards/creation/dashboards.json';
+import genericTextWidgets from '../../../fixtures/dashboards/creation/widgets/genericText.json';
+import metricsGraphWidget from '../../../fixtures/dashboards/creation/widgets/metricsGraphWidget.json';
+import metricsGraphDoublecWidget from '../../../fixtures/dashboards/creation/widgets/dashboardWithTwometricsGraphWidget.json';
+
+before(() => {
+  cy.startWebContainer();
+  cy.updateCentreonConfig();
+  cy.executeCommandsViaClapi(
+    'resources/clapi/config-ACL/dashboard-metrics-graph.json'
+  );
+  const apacheUser = Cypress.env('WEB_IMAGE_OS').includes('alma')
+    ? 'apache'
+    : 'www-data';
+  cy.execInContainer({
+    command: `su -s /bin/sh ${apacheUser} -c "/usr/bin/env php -q /usr/share/centreon/cron/centAcl.php"`,
+    name: Cypress.env('dockerName')
+  });
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/dashboards?'
+  }).as('listAllDashboards');
+  cy.intercept({
+    method: 'POST',
+    url: `/centreon/api/latest/configuration/dashboards/*/access_rights/contacts`
+  }).as('addContactToDashboardShareList');
+  cy.intercept({
+    method: 'GET',
+    url: /\/api\/latest\/monitoring\/dashboard\/metrics\/performances\/data\?.*$/
+  }).as('performanceData');
+});
+
+beforeEach(() => {
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/internal.php?object=centreon_topology&action=navigationList'
+  }).as('getNavigationList');
+  cy.intercept({
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/dashboards?'
+  }).as('listAllDashboards');
+  cy.intercept({
+    method: 'POST',
+    url: `/centreon/api/latest/configuration/dashboards/*/access_rights/contacts`
+  }).as('addContactToDashboardShareList');
+  cy.intercept({
+    method: 'PATCH',
+    url: `/centreon/api/latest/configuration/dashboards/*`
+  }).as('updateDashboard');
+  cy.intercept({
+    method: 'GET',
+    url: /\/api\/latest\/monitoring\/dashboard\/metrics\/performances\/data\?.*$/
```

Comment:
is there a reason why the url is not between ` here and not elsewhere?

Reply:
in fact i tried to implement the same thing but it doesn't work since the api is very complicated